index.html

```diff
@@ -68,7 +68,8 @@ <h2 class="headlinejob">I'm a Softwere Developer</h2>
          <p class="work-dicreption">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's
          standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
          <img class="languages-Fwork" src="img/Tags.svg" alt="languages used at recent work (css,html,bootstrap,ruby)">
```

Comment:
- [ ] Please make sure to update the main page so that the projects section is created dynamically using the information stored in that JavaScript array. All of the HTML in that section should be created when the page loads as requested in the project requirements.